doc/src/manual/style-guide.md

```diff
@@ -412,4 +412,4 @@ julia> h(1)
 ```
 
 Thus, use `Int` literals when possible, with `Rational{Int}` for literal non-integer numbers,
```

Comment:
I'd strike the whole dependent clause and if anything add `Thus, to make it easier to re-use your code, ...` or just leave it out. The `why` was already explained above.